scout/constants/gene_tags.py

```diff
@@ -0,0 +1,12 @@
+INHERITANCE_MODELS = (
```

Comment:
There appears to be a constant in the panel controller with the same name? Perhaps this should be CUSTOM_INHERITANCE_MODELS?

Reply:
Also perhaps a couple of comments describing the difference to GENETIC_MODELS and matching OMIM inheritance models?

Reply:
You are right, I'll change to constant name into GENE_CUSTOM_INHERITANCE_MODELS!

Reply:
I don't have any official reference to OMIM for these models. The idea is that the should be custom, and in fact this is a list compiled by @4WGH. I don't know if there is an official documentation in OMIM and most importantly if it corresponds to these models.